gateway/cli/src/main.rs

```diff
@@ -110,14 +117,18 @@ async fn main() {
     }
 }
 
-pub async fn call<P>(url: String, endpoint: String, payload: P)
+pub async fn call<P>(password: String, url: String, endpoint: String, payload: P)
 where
     P: Serialize,
 {
     let client = reqwest::Client::new();
 
     let response = client
         .post(format!("{}{}", url, endpoint))
+        .header(
+            reqwest::header::AUTHORIZATION,
+            format!("Bearer {}", password),
+        )
```

Comment:
You can use [bearer_auth](https://docs.rs/reqwest/0.11.12/reqwest/struct.RequestBuilder.html#method.bearer_auth) instead imo.